Tasks/TwineAuthenticateV0/Authentication.ts

```diff
@@ -0,0 +1,133 @@
+import * as pkgLocationUtils from "utility-common/packaging/locationUtilities";
```

Comment:
It feels like there's a fair amount of stuff in here that belongs in some common place